README.md

```diff
@@ -189,6 +190,20 @@ By default, the packages are looked for in the `vendor-bin` directory. The locat
 }
 ```
 
+### Reduce clutter
```

Comment:
This is missing an entry in the ToC

Reply:
Yup. I have added it along with another entry that was missing ;) .